spikingjelly/activation_based/functional/forward.py

```
@@ -251,6 +259,8 @@ def seq_to_ann_forward(
         tensor tuple，则分别恢复每个 tensor 的时间维和批量维
     :rtype: Union[torch.Tensor, tuple[torch.Tensor, ...]]
 
+    :raises ValueError: 当 ``x_seq`` 为空tuple，或tuple中tensor的维数少于2，或tuple中tensor的 ``[T, batch_size]`` 前两维不一致时抛出
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)